src/check_release.rs

```diff
@@ -150,6 +150,20 @@ fn print_triggered_lint(
                 Ok(())
             })?;
         }
+
+        if let Some(hint_template) = semver_query.witness.hint_template.as_deref() {
+            let message = config
+                .handlebars()
+                .render_template(hint_template, &pretty_result)
+                .context("Error instantiating witness hint template.")?;
+
+            config.log_info(|config| {
+                config.shell_note("the following downstream code would break:")?;
```

Comment:
Let's make sure the user can't accidentally think we've already *found* some specific downstream code that we're complaining about, and instead we're presenting a hypothetical piece of code that would be broken:
```suggestion
                config.shell_note("downstream code similar to the following would break:")?;
```